scrapers/covid-19/govScrapers/getIsrael.js

```diff
@@ -21,30 +22,43 @@ const params = {
  */
 const parseData = (data) => {
 	const dateRegex = /\d+-\d+-\d+/g;
-	const patientsPerDay = data[2].data;
-	const deadPatientsPerDay = data[3].data;
-	const recoveredPerDay = data[4].data.splice(20, data[4].data.length - 1);
-	const testsPerDay = data[5].data.splice(30, data[5].data.length - 1);
+	const patientsPerDay = data[1].data;
+	const deadPatientsPerDay = data[2].data;
+	const recoveredPerDay = data[3].data.splice(20, data[3].data.length - 1);
+	const testsPerDay = data[4].data.splice(30, data[4].data.length - 1);
 	const timeline = patientsPerDay.map((elem, index) => ({
 		date: elem.date.match(dateRegex)[0],
 		newHospitalized: elem.new_hospitalized,
-		totalhospitalized: elem.Counthospitalized,
```

Comment:
keep this property for anyone who may have been using it before, it would be a breaking change.

Reply:
why was it `totalhospitalized` anyway

Reply:
I'm sure no one is using this yet... It was just added yesterday.